nilearn_sandbox/plotting/papaya.py

```diff
@@ -0,0 +1,56 @@
+import base64
+import tempfile
+import nibabel
+import numpy as np
+import os
+from nilearn._utils import check_niimg
+
+
+def _get_64(niimg):
+    f = check_niimg(niimg)
+    data = f.get_data().astype(float)
+    data = data + data.min()
+    data = data / data.max() * 254.
+    data = data.astype(np.uint8)
+    f = nibabel.Nifti1Image(data, f.get_affine())
+    _, filename = tempfile.mkstemp(suffix='.nii.gz')
+    f.to_filename(filename)
+    with open(filename, 'br') as f:
+        b64 = base64.b64encode(f.read())
+    os.unlink(filename)
+    return b64.decode('utf-8')
+
+
+def papaya_viewer(maps_niimg, output_file=None):
+    import tempita
```

Comment:
This is all part of the sandbox logic, you can see the README file for information. The goal of this package is to add functions regardless of there dependencies. By doing so, the dependency must not block the import of the package, this is why it is local. And I didn't put it as a strong dependency otherwise, if this sandbox is successful, we'll have a lot of dependencies.

On the other hand, I should fail more gracefully and I totally should advertise somewhere that this functionality requires tempita.